templates/nginx.conf.erb

```diff
@@ -1,4 +1,4 @@
-user www-data;
+user <%= @user || 'www-data' %> <%= @group || 'www-data' %>;
```

Comment:
the defaults here shouldn't be necessary as they're defined on the params in the manifest, it may also hide a misconfiguration if the params were unintentionally set to an empty string.